Stock_Alert/errorhandling.py

```diff
@@ -0,0 +1,32 @@
+import requests
+import json
+
+class ErrorHandling:
+    def __init__(self, endpoint):
+        self.endpoint = endpoint
+
+
+    def get_hold_of(self, **kwargs):
+        try:
+            response = requests.get(self.endpoint, params=kwargs)
+            response.raise_for_status()
+            return response
+        # When using a wrong API key for the API endpoint of Alpha Vantage,
+        # it currently returns still the status code 200.
+        # The following workaround catches the wrong input as long as response.content contains the error message.
+        except ConnectionError as c:
+            print("An error occurred during your request: ", c)
+            quit()
+        except requests.exceptions.RequestException as e:
+            if response.status_code == 200:
+                result = response.content
```

Comment:
```suggestion
                result = response.content.json()
```

and you don't need the next line then

Reply:
I removed the error handling for the edge case here and added a small work around instead in main.py as you suggested.